src/IO.Ably.Android/WebsocketTransport.cs

```
@@ -188,8 +188,10 @@ private void socket_DataReceived(object sender, DataReceivedEventArgs e)
             {
                 try
                 {
+#if MSGPACK
```

If MSGPACK is disabled add a log message because that shouldn't be happening. It's always good to know.

---

👍

---

Actually, we should throw an exception as MsgPack is not supported surely?

---

I will fix up a version that throws an exception, for review.

---

actually not sure about this, see comment elsewhere